app/views/dashboards/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :javascript_includes do %>
+  <%= javascript_include_tag "dashboard_poller" %>
```

Comment:
Ignore the above comment. I think the entire way the dashboard refresh is configured and triggered needs to be changed so is a separate task all to itself